src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -94,7 +94,7 @@ define(function (require, exports, module) {
             var mtime = new Date(stats.mtime);
 
             var options = {
-                isFile: stats.isFile(),
```

Comment:
When the `stats` object comes back over the `MessageChannel`, it won't have the prototype functions.  You can't pass functions across a channel.